.github/workflows/ci.yml

```diff
@@ -93,6 +96,10 @@ jobs:
         E2E_TEST_ESS_POD: ${{ secrets.E2E_TEST_ESS_PROD_POD }}
```

Comment:
Could we fix up the env variable order, so that it is consistent accross envs?

It's no big deal, but makes it easier to check we're consistent and there's no mistake. Possibly, doing an "order alphabetically" on such lists could be a good move.

(here in TEST_ESS, IDP_URL comes before POD)

Potentially, we might want to align the secret names too, that is, rename  E2E_TEST_ESS_PROD_IDP_URL to E2E_TEST_ESS_IDP_URL...

(a bit tedious I imagine, so not necessary right now)

Reply:
I'll create a task in the backlog to reorder the variables. The naming pattern can also be reconsidered, but the presence of "PROD" is intentional, since `pod.inrupt.com` is our prod environment :). Bet we could consider changing `DEV_NEXT` into `ESS_DEV_NEXT`, even though `DEV_NEXT` is ESS-only, so it may make more sense to change `ESS_PROD` to just `PROD` (that's assuming we'll never have a CSS prod environment)

Reply:
I was looking at it from the point of view of the secret naming convention as it stands: **E2E_TEST_ESS_POD** vs secrets.E2E_TEST_ESS_**PROD**_POD.

I think secret and variable names should align.
I also think that you are correct, arguably, keeping the `PROD` part makes more sense.

But on second thoughts, more importantly, it might be a mistake to adapt our code and tests for this.
We should probably have only one series of tests that take standard environment variables and is run with the series of PROD variables, then the series of DEV_NEXT variable and so on...

So really, where we should go is having this in CI:
```bash
E2E_TEST_POD: ${{ secrets.E2E_TEST_ESS_PROD_POD }}
```

And adapt the workflow to run for each environment.

Then in our code, we'll keep it simple and run with parameterizable TEST_POD, TEST_IDP_URL...

Reply:
(probably not for this PR depending on prioritisation and urgency)